### PR TITLE
Update maintainers

### PR DESCRIPTION
**Pull request by adityapande-1995**
Updates maintainers in the contained packages.

### CODE REVIEW

**Review by ahcorde — APPROVED** [review]

**Review by ahcorde — CHANGES_REQUESTED** [review]
Do you mind to add me as a maintaner too ? and also include us in the CODEOWNERS file

File: ros_gz_sim_demos/package.xml [hunk 1]
```diff
@@ -4,6 +4,8 @@
   <description>Demos using Gazebo Sim simulation with ROS.</description>
   <license>Apache 2.0</license>
   <maintainer email="louise@openrobotics.org">Louise Poubel</maintainer>
```
Comment (ahcorde) [inline ros_gz_sim_demos/package.xml]:
```suggestion
```

**Review by ahcorde — CHANGES_REQUESTED** [review]
can you remove louise from all package.xml ? and add her as an author?

File: ros_gz_sim_demos/package.xml (hunk 1 above)
Comment (adityapande-1995) [reply ros_gz_sim_demos/package.xml]: Done

**Review by azeey — APPROVED** [review]

**Review by ahcorde — APPROVED** [review]

**Pull request merged**